tests/components/bond/test_cover.py

```diff
@@ -0,0 +1,113 @@
+"""Tests for the Bond cover device."""
+import logging
+
+from bond import BOND_DEVICE_TYPE_MOTORIZED_SHADES
+
+from homeassistant.components.bond.utils import BondDevice
+from homeassistant.components.cover import DOMAIN as COVER_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_CLOSE_COVER,
+    SERVICE_OPEN_COVER,
+    SERVICE_STOP_COVER,
+)
+from homeassistant.helpers.entity_registry import EntityRegistry
+
+from .common import setup_platform
+
+from tests.async_mock import patch
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def test_entity_registry(hass):
+    """Tests that the devices are registered in the entity registry."""
+
+    with patch(
+        "homeassistant.components.bond.utils.get_bond_devices",
+        return_value=[
+            BondDevice(
+                "device-1",
+                {"name": "name-1", "type": BOND_DEVICE_TYPE_MOTORIZED_SHADES},
+            )
+        ],
+    ):
+        await setup_platform(hass, COVER_DOMAIN)
+
+    registry: EntityRegistry = await hass.helpers.entity_registry.async_get_registry()
+    assert [key for key in registry.entities.keys()] == ["cover.name_1"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)